src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -115,18 +115,12 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
             var files = await fileProvider.GetFilesAsync(token);
             logger.WriteLine(MigrationStrings.Process_CountOfFilesToCheck, files.Count());
 
+            ChangedFiles changedFiles = null;
```

Comment:
When we interact with the source control system via the VS APIs, we do it as follows:
* tell VS we are starting a batch of operations
* check out and modify one or more files
* tell VS we have finished the batch.

I've moved all of the file-changing operations to a single method to make it simpler to put them all inside a batch (the batching operations are added in a later commit).